Program.cs

```diff
@@ -13,23 +13,16 @@ static void Main()
 		{
 			CultureInfo.DefaultThreadCurrentCulture = CultureInfo.InvariantCulture;
 
-			try
-			{
-				var settings = Settings.Load(SettingsFile);
-
-				using (var nativeHelper = new NativeHelper())
-				{
-					Application.EnableVisualStyles();
-					Application.SetCompatibleTextRenderingDefault(false);
-					Application.Run(new MainForm(nativeHelper, settings));
-				}
+			var settings = Settings.Load(SettingsFile);
 
-				Settings.Save(settings, SettingsFile);
-			}
-			catch (Exception ex)
+			using (var nativeHelper = new NativeHelper())
 			{
-
+				Application.EnableVisualStyles();
+				Application.SetCompatibleTextRenderingDefault(false);
+				Application.Run(new MainForm(nativeHelper, settings));
```

Comment:
Weird it doesn't do that by default, mine does (vs 2015 community)
here is my settings: 
![image](https://cloud.githubusercontent.com/assets/994534/19406000/b37025f2-92c1-11e6-9b58-d6670ea56c4b.png)